licenseclassifier.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: licenseclassifier
+  version: "2.0.0"
+  epoch: 0
+  description: The license classifier is a library and set of tools that can analyze text to determine what type of license it contains.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/google/licenseclassifier
+      tag: v${{package.version}}
+      expected-commit: e6a9bb99b5a6f71d5a34336b8245e305f5430f99
+      cherry-picks: |
+        main/0181809d0f4bf2ee94d3fb19906fbf7be3801a5f: fix license database embedding
+
+  - name: Generate the license database
+    runs: |
+      touch licenses/licenses.db
+      go run tools/license_serializer/license_serializer.go -output licenses
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./tools/identify_license
+      output: licenseclassifier
+
+update:
+  enabled: true
+  github:
+    identifier: google/licenseclassifier
+    strip-prefix: v
```

Comment:
I see the repo has tags w/ "pre" and "alpha" that we probably want to ignore

Reply:
Good catch! Thanks!